SecurityManager/Core/Web.config

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+    <configSections>
+        <sectionGroup name="remotion.data.domainObjects"
+                type="Remotion.Data.DomainObjects.Configuration.DomainObjectsConfiguration, Remotion.Data.DomainObjects">
+            <section name="storage"
+                    type="Remotion.Data.DomainObjects.Persistence.Configuration.StorageConfiguration, Remotion.Data.DomainObjects" />
+        </sectionGroup>
+    </configSections>
+
+    <remotion.data.domainObjects xmlns="http://www.re-motion.org/Data/DomainObjects/Configuration/2.1">
+        <storage defaultProviderDefinition="SecurityManager">
+            <providerDefinitions>
+                <add type="Remotion.Data.DomainObjects::Persistence.Rdbms.RdbmsProviderDefinition"
+                        name="SecurityManager"
+                        factoryType="Remotion.SecurityManager::Persistence.SecurityManagerSqlStorageObjectFactory"
+                        connectionString="RemotionSecurityManagerWebClient" />
+            </providerDefinitions>
+        </storage>
+    </remotion.data.domainObjects>
+
+    <connectionStrings>
+        <add name="RemotionSecurityManagerWebClient"
```

Comment:
Please test, I'm not sure if we have a string-dependency on the name somewhere, but if we can rename this and not get a changed DB name etc, we should drop the `WebClient` part.